src/renderer/Menu.tsx

```diff
@@ -69,14 +140,47 @@ const Menu: React.FC<MenuProps> = function ({ t, error }: MenuProps) {
 					<>
 						<span className={classes.waiting}>{t('game.waiting')}</span>
 						<CircularProgress color="primary" size={40} />
-						<button
-							className={classes.button}
-							onClick={() => {
-								ipcRenderer.send(IpcMessages.OPEN_AMONG_US_GAME);
-							}}
+						<ButtonGroup variant="contained" ref={anchorRef}>
```

Comment:
I can see you aren't a designer as well. Idk if I'm a fan of the current font size of the button/ of the button at all atm. 
Since it's quite big. Idk if it looks better to make a selectbox above the button and keep the button small. 
I'm not sure if that will look better tho.